README.md

```diff
@@ -66,6 +68,34 @@ Follow [this guide](./test/spectests/README.md) for using the tool.
 
 This is the unit tests suite of Fizzy.
 
+## Special notes about floating point
+
+### Exceptions
+
+Certain floating point operations can emit *exceptions* as defined by the IEEE 754 standard.
+(Here is a [summary from glibc](https://www.gnu.org/software/libc/manual/html_node/FP-Exceptions.html)).
+It is however up to the language how these manifest, and in C/C++ depending on the settings they can result in
+a) setting of a flag; b) in *traps*, such as the `SIGFPE` signal.
+
+Fizzy does not manipulate this setting, but expects it to be left at option a) of above, which is the default.
+In glibc this can be controlled via the [`feenableexcept` and `fedisableexcept` functions](https://www.gnu.org/savannah-checkouts/gnu/libc/manual/html_node/Control-Functions.html).
```

Comment:
```suggestion
In GNU C Library this can be controlled via the [`feenableexcept` and `fedisableexcept` functions](https://www.gnu.org/savannah-checkouts/gnu/libc/manual/html_node/Control-Functions.html).
```